### DISCUSSION TITLE
ModuleNotFoundError: No module named 'torchvision' (but it is installed).

### DISCUSSION THREAD

**Discussion by KTFish** [mrdbourke/pytorch-deep-learning#464] [Q&A]
Hello there!
I have a problem with torchvision module. I have installed both pytorch and torchvision. VS Code do not underlines the torchvision import so it look everything should be fine but I sitll get this error:
```
Traceback (most recent call last):
  File "PATH_TO_FILE", line 3, in <module>
    from torchvision.datasets import CIFAR100
ModuleNotFoundError: No module named 'torchvision'
```
Do anyone know what could help?

**Comment by mrdbourke** [comment]
Hi @KTFish ,

Have you tried to reinstall `torchvision`? 

Or perhaps reinstall `torch` from scratch?

See here: https://pytorch.org/get-started/locally/ 

---

Otherwise have you tried just loading your environment in the terminal and then starting up Python and trying to import `torchvision`? Does that work?

**Reply by KTFish** [reply]
Thanks, I tried reinstalling both torchvision and torch. I figured out that it works in VS Studio in Jupyter Notebook mode (when pressing `Shift + Enter` in .py file), however, in normal .py files I still have the same error. I use the same environment in both cases.
